projects/packages/my-jetpack/src/products/class-product.php

```diff
@@ -92,6 +92,7 @@ public static function get_info() {
 			'long_description'         => static::get_long_description(),
 			'features'                 => static::get_features(),
 			'status'                   => static::get_status(),
+			'pricing_for_ui'           => static::get_pricing_for_ui(),
```

Comment:
Why add `for_ui` suffix? I think just `pricing` makes sense, not?

Reply:
Reasons explained here: p1643837371383059-slack-C02TQF5VAJD

Reply:
sorry, thanks for the link